test/uk/gov/hmrc/transitmovementsvalidator/data/cc015c-generated-from-json-schema.json

```diff
@@ -0,0 +1,1261 @@
+{
+  "#": "String",
```

Comment:
Can we test the validator against the Json I have in the converter?

Reply:
(See cc015c-2.json on the ticket CTCP-319)

Reply:
Adding this in the unit test